tests/test_infer.py

```diff
@@ -466,16 +466,16 @@ def test_toy_calculator(tmpdir, hypotest_args):
     )
 
 
-def test_fixed_poi(tmpdir, hypotest_args):
+def test_fixed_poi(hypotest_args):
     """
     Check that the return structure of pyhf.infer.hypotest with the
     addition of the return_expected keyword arg is as expected
     """
 
-    _, _, pdf = hypotest_args
-    pdf.config.param_set('mu').suggested_fixed = [True]
+    test_poi, data, model = hypotest_args
+    model.config.param_set("mu").suggested_fixed = [True]
     with pytest.raises(pyhf.exceptions.InvalidModel):
-        pyhf.infer.hypotest(*hypotest_args)
+        pyhf.infer.hypotest(test_poi, data, model)
```

Comment:
Changing this still to make it more clear that the unpacked `model` is what is being passed in, as otherwise this might be confusing at first glance.